### PR TITLE
3 Up component

### LINKED ISSUE #142: Build 3 Up component

### PR DESCRIPTION
**Pull request by gvn**
closes #142

### CODE REVIEW

**Comment by gvn** [comment]
You can review this at: [http://127.0.0.1:1818/style-guide](http://127.0.0.1:1818/style-guide)

File: app/react/components/three-up/three-up.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    .copy {
+      font-size: 1.5rem;
+    }
+  }
+
+  .icon {
+    border-radius: 50%;
+    background: $summit-teal;
+    width: 132px;
+    height: 132px;
```
Comment (alanmoo) [inline app/react/components/three-up/three-up.scss]: Should we use 13.2 rem to be consistent/flexible in the future? Or maybe em so we can set a parent size on the component if necessary?

**Comment by alanmoo** [comment]
This looks strange on viewports around 500px wide- maybe you need some `col-md` classes?

**Comment by gvn** [comment]
@alanmoo Ok, did some fixups. R?

**Comment by alanmoo** [comment]
Did this not pass travis?

**Comment by gvn** [comment]
It passed `npm test` locally, but yeah I probably should have waited for Travis.

**Pull request merged**